includes/Core/Assets/Assets.php

```diff
@@ -481,6 +481,15 @@ private function get_assets() {
 					),
 				)
 			),
+			new Script(
+				'googlesitekit-datastore-forms',
+				array(
+					'src'          => $base_url . 'js/googlesitekit-datastore-forms.js',
```

Comment:
This will need a related webpack entry point added as well, eg. https://github.com/google/site-kit-wp/blob/develop/webpack.config.js#L96